Doc/library/os.rst

```diff
@@ -1698,6 +1699,19 @@ features:
    Return a bytestring representing the current working directory.
 
 
+.. function:: get_current_dir_name()
+
+    Return a string representing the current working directory taking into
+    consideration the users ``PWD`` environment variable if it exists. This is
+    opposed to :func:`getcwd()` which dereferences symlinks in the path. This
+    function is identical to :func:`getcwd()` on systems that do **not**
+    support the ``PWD`` environment variable.
+
+    .. availability:: Unix.
```

Comment:
This function is available on all platforms.

Reply:
Actually, while this is undoubtably an ancient link ( https://www.gnu.org/software/gnulib/manual/html_node/get_005fcurrent_005fdir_005fname.html ) I do not believe this function is available in all POSIX environments. It may be common in GNU runtime environments.

FYI: I just checked AIX and found the routine name "exists" in AIX <unistd.h>, but there is no documentation. So, it seems it may be visible on AIX, even if they have never taken the time to a is adding dd it to the getcwd() manual page.

Confusion #1 in adding this